setup.sh

```diff
@@ -0,0 +1,25 @@
+!/bin/bash
+
+Run the go_installer script with the parent directory as an argument
+./go_installer.sh ../
+echo "Installed go" 
+go version
+sudo apt update -y
```

Comment:
I think apt is not default package manager for the all the distros we support. for example it may not work for rhel out of the box.